### PR TITLE
UI Component - Alert

### LINKED ISSUE #106: UI Component - Create Alert component

### LINKED ISSUE #107: Alert component: creation

### PR DESCRIPTION
**Pull request by Enessetere**
closes #106 
closes #107

### CODE REVIEW

File: src/app/components/Alert.ts [hunk 1]
```diff
@@ -0,0 +1,58 @@
+import { AlertType } from '../../Enums/AlertEnum';
+import { Render } from '../utils/Render';
+
+export class Alert {
+  private static alert: HTMLElement;
+
+  /**
+   * Creates and returns Alert component
+   * Usage: Alert.createElement()
```
Comment (vieraboschkova) [inline src/app/components/Alert.ts:9]: Very handy! 🎆

File: src/app/components/Alert.ts [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+        this.alert.style.borderColor = 'red';
+        break;
+      default:
+        this.alert.style.borderColor = 'white';
+        break;
+    }
+    const icon = Render.elementFactory('img', {
+      className: 'alert__icon',
+      src: `./static/images/ui/${alertType}.png`,
+      alt: `${alertType}-alert`,
```
Comment (vieraboschkova) [inline src/app/components/Alert.ts:49]: 👍

**Review by vieraboschkova — APPROVED** [review]
Good job!

**Review by synowa — APPROVED** [review]
Looks good 🚀

**Pull request merged**